### DISCUSSION TITLE
zebra plugin printing very very small label

### DISCUSSION THREAD

**Discussion by anonkwe** [inventree/InvenTree#4515] [Q&A]

**Comment by wolflu05** [comment]
Hello @anonkwe , please provide more information what your config, results and expected behaviors are. And please post that at their repository, as this is not directly related to inventree core.

**Reply by anonkwe** [reply]
@wolflu05 sorry about that, couldn't delete this.. maybe if you can check on this link and tell me if you have any idea. I sure would appreciate it

 [](https://github.com/inventree/InvenTree/issues/4516)
